src/controller/python/chip/commissioning/commissioning_flow_blocks.py

```diff
@@ -0,0 +1,182 @@
+        dac = await self._devCtrl.SendCommand(device.node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.CertificateChainRequest(
+            certificateType=1
+        ))
+
+        self._logger.info("Getting CertificateChain - PAI")
+        pai = await self._devCtrl.SendCommand(device.node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.CertificateChainRequest(
+            certificateType=2
+        ))
+
+        self._logger.info("Getting CSR Nonce")
+        csr_nonce = await self._credential_provider.get_csr_nonce()
+
+        self._logger.info("Getting OpCSRRequest")
+        csr = await self._devCtrl.SendCommand(device.node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.CSRRequest(
+            CSRNonce=csr_nonce
+        ))
+
+        self._logger.info("Getting device certificate")
+        commissionee_credentials = await self._credential_provider.get_commissionee_credentials(
+            commissioning.GetCommissioneeCredentialsRequest(
+                dac=dac, pai=pai,
+                attestation_challenge=attestation_nonce,
+                attestation_elements=attestation_elements.attestationElements,
+                attestation_signature=attestation_elements.signature,
+                csr_challenge=csr_nonce,
+                csr_elements=csr.NOCSRElements,
+                csr_signature=csr.attestationSignature,
+                vendor_id=device_info.vendorID,
+                product_id=device_info.productID))
+
+        self._logger.info("Adding Trusted Root Certificate")
+        response = await self._devCtrl.SendCommand(device.node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.AddTrustedRootCertificate(
+            rootCertificate=commissionee_credentials.rcac
+        ))
+
+        self._logger.info("Adding Operational Certificate")
+        response = await self._devCtrl.SendCommand(device.node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.AddNOC(
+            NOCValue=commissionee_credentials.noc,
+            ICACValue=commissionee_credentials.icac,
+            IPKValue=commissionee_credentials.ipk,
+            caseAdminSubject=commissionee_credentials.case_admin_node,
+            adminVendorId=commissionee_credentials.admin_vendor_id
+        ))
+        if response.statusCode != 0:
+            raise commissioning.CommissionFailure(repr(response))
+
+        self._logger.info("Setting fabric label")
+        response = await self._devCtrl.SendCommand(device.node_id, commissioning.ROOT_ENDPOINT_ID, Clusters.OperationalCredentials.Commands.UpdateFabricLabel(
+            label=parameter.fabric_label
+        ))
+        if response.statusCode != 0:
+            raise commissioning.CommissionFailure(repr(response))
+
+        self._logger.info(f"Device peer id: {commissionee_credentials.fabric_id:016x}:{commissionee_credentials.node_id:016x}")
```

Comment:
Let's get the compressed fabric ID, not just the bare fabric ID.

ALso, let's not call it the "peer id". 

We should should log this before AddNOC as:

`f"Commissioning FabricID {commissionee_credentials.fabric_id:016X}, Compressed Fabric ID {commissionee_credentials.compressed_fabric_id:016X}, Node ID {commissionee_credentials.node_id:016X}"`

This way it is unambiguous which parts are what. We should also make sure (as I illustrate) to use uppercase hex like the SDK logging does.